plugins/org.wso2.developerstudio.bpmn.extensions/src/org/wso2/developerstudio/bpmn/extensions/ExtensionConstants.java

```diff
@@ -0,0 +1,11 @@
+package org.wso2.developerstudio.bpmn.extensions;
+
+public class ExtensionConstants {
+
+    public static final String WSO2_TASK_LIB_LOCATION = "libs/org.wso2.developerstudio.bpmn.extensions-3.6.0-SNAPSHOT.jar";
+
```

Comment:
Don't hard code this value. We need to externalize this or pick dynamically at runtime. .